source/security/enterprise-auth.txt

```diff
@@ -326,6 +326,12 @@ see the corresponding syntax.
 
    .. tab:: Connection String
       :tabid: mongodb-azure-imds-connection-string
+
+      To specify Kubernetes OIDC as the authentication mechanism, set the following 
+      options in your connection string:
+
+      - ``authMechanism``: Set to ``MONGODB-OIDC``.
+      - ``authMechanismProperties``: Set to ``ENVIRONMENT:k8s``.
```

Comment:
These should specify the correct environment.

Reply:
Good catch, thank you!